src/Input/Input.variants.js

```diff
@@ -0,0 +1,79 @@
+import { BORDERS } from '../assets/core'
+import colors from '../assets/colors'
+import typography from '../Typography/variants'
+
+const DEFAULT_STYLE = {
+  ...typography.inputLabel,
+  height: '40px',
```

Comment:
Please extract this into a variable in `assets`, for example:

```
import sizes from '../assets/size'

const DEFAULT_STYLE = {
  ...typography.inputLabel,
  height: sizes.inputHeight

```

Reply:
Done